mnefun/_mnefun.py

```diff
@@ -186,6 +187,38 @@ class Params(Frozen):
         High-pass transition band.
     movecomp : str | None
         Movement compensation to use. Can be 'inter' or None.
+    sss_type : str
+        signal space separation method. Must be either 'maxfilter' or 'python'
+    int_order : int
+        Order of internal component of spherical expansion.
+    ext_order : int
+        Order of external component of spherical expansion.
+    tsss_dur : float | None
+        Apply spatiotemporal SSS with specified buffer duration
+        (in seconds). Elekta's default is 10.0 seconds in MaxFilter™ v2.2.
+        Spatiotemporal SSS acts as implicitly as a high-pass filter where the
+        cut-off frequency is 1/st_dur Hz. For this (and other) reasons, longer
+        buffers are generally better as long as your system can handle the
+        higher memory usage. To ensure that each window is processed
+        identically, choose a buffer length that divides evenly into your data.
+        Any data at the trailing edge that doesn't fit evenly into a whole
+        buffer window will be lumped into the previous buffer.
+    st_correlation : float
+        Correlation limit between inner and outer subspaces used to reject
+        ovwrlapping intersecting inner/outer signals during spatiotemporal SSS.
+    trans_to : str | array-like, shape (3,) | None
+        The destination location for the head. Can be ``None``, which
+        will not change the head position, or a string path to a FIF file
+        containing a MEG device<->head transformation, or a 3-element array
+        giving the coordinates to translate to (with no rotations).
+        For example, ``destination=(0, 0, 0.04)`` would translate the bases
+        as ``--trans default`` would in MaxFilter™ (i.e., to the default
+        head location).
```

Comment:
I don't think this one is actually correct --  we only allow `'median'` and `'default'`, right? Whatever we allow, we should allow the same set of options for the `mne-python` version and the `MaxFilter` verison, and they should behave the same way.